flycheck.el

```diff
@@ -2130,11 +2130,14 @@ into the verification results."
   (insert-button (symbol-name checker)
                  'type 'help-flycheck-checker-doc
                  'help-args (list checker))
-  (when (with-current-buffer buffer (flycheck-disabled-checker-p checker))
+  (when (buffer-local-value (flycheck-disabled-checker-p checker) buffer)
```

Comment:
:clap:

Reply:
Actually, that would have worked with the `flycheck-disabled-checkers` variable, but it doesn't with the predicate :sweat:.